include/CL/sycl.hpp

```diff
@@ -1,28 +1,12 @@
 /*
- * This file is part of hipSYCL, a SYCL implementation based on CUDA/HIP
+ * This file is part of AdaptiveCpp (formerly hipSYCL, Open SYCL), an
+ * implementation of SYCL and C++ standard parallelism for CPUs and GPUs
  *
- * Copyright (c) 2018-2020 Aksel Alpay
+ * Copyright (c) 2018-2024 Aksel Alpay and contributors
```

Comment:
What is the "year" policy going forward?

- Linux Foundation suggests not indicating it at all: https://www.linuxfoundation.org/blog/blog/copyright-notices-in-open-source-software-projects
- FSF suggest listing "the year(s) in which you finished preparing the release": https://www.gnu.org/licenses/gpl-howto.en.html
- GNU suggests listing "each year in which you have made nontrivial changes to the package" (updating all files at once): https://www.gnu.org/prep/maintain/html_node/Copyright-Notices.html

- Curl decided to drop years recently, since they are (allegedly) not required for copyright to hold while being a time sink: https://daniel.haxx.se/blog/2023/01/08/copyright-without-years/
- GROMACS used scripts to update the year every time the file is modified, but a while ago switched to having open-ended "(year-of-file-creation)-" interval (which, now that I read a bit more about it, seems to be not compliant with the US Copyright notice requirement): https://gitlab.com/gromacs/gromacs/-/issues/4322.

Reply:
That's a good point. If it's legal to drop (it seems), I would be fine with following what Linux Foundation recommends, as this seems less work for us.